docs/sandbox-permissions.rst

```diff
@@ -273,12 +298,11 @@ depends on the kernel/fstab configuration and cannot be pre-determined.
    and these have no permission checks. This unfortunately affects e.g. the X
    server and the session bus which listens to abstract sockets by default. A
    secure distribution should disable these and just use regular sockets.
-.. [#f3] Except for ``/app, /bin, /boot, /efi, /etc, /lib, /lib32, /lib64, /proc, /root, /run, /sbin, /tmp, /usr, /var``
-.. [#f4] This does not include access to folders under ``~/.var/app`` except the application's own
-.. [#f5] Except ``/app, /dev, /etc, /lib, /lib32, /lib64, /proc, /root, /run/flatpak, /run/host, /sbin, /usr``
-.. [#f6] The arbitrary path includes all its subfolders and subfiles if any.
-.. [#f7] ``xdg-{cache, config, data}`` binds mount the paths from host to the per-app sandbox directory.
-   Inside the sandbox ``$XDG_CACHE_HOME, $XDG_CONFIG_HOME and $XDG_DATA_HOME`` is set to
-   ``$HOME/.var/app/<app-id>/{cache, config, data}``. So this permission is not needed
-   unless access to the host directory, bind mounted to
-   ``$HOME/.var/app/<app-id>/{cache, config, data}`` is desired.
+.. [#f3] ``xdg-{cache, config, data}`` bind mounts the paths from host to the per-app sandbox directory.
+   Inside the sandbox ``$XDG_CACHE_HOME``, ``$XDG_CONFIG_HOME`` and ``$XDG_DATA_HOME`` is set to
+   ``$HOME/.var/app/app-id/{cache, config, data}`` respectively. So for example, ``xdg-data/applications`` ie.
+   ``$XDG_DATA_HOME/applications`` on host is bind mounted to ``$HOME/.var/app/app-id/data/applications``
+   (inside the sandbox this is ``$XDG_DATA_HOME/applications``).
+   Additionally it'll have two mount points - one expanded to
+   ``$XDG_DATA_HOME/applications`` from the host and another to the
+   sandbox's ``$XDG_DATA_HOME/applications`` ie. ``$HOME/.var/app/app-id/data/applications``.
```

Comment:
```suggestion
   ``$HOME/.var/app/<app-id>/{cache, config, data}`` respectively. So for example, ``xdg-data/applications`` ie.
   ``$XDG_DATA_HOME/applications`` on host is bind mounted to ``$HOME/.var/app/<app-id>/data/applications``
   (inside the sandbox this is ``$XDG_DATA_HOME/applications``).
   Additionally it'll have two mount points - one expanded to
   ``$XDG_DATA_HOME/applications`` from the host and another to the
   sandbox's ``$XDG_DATA_HOME/applications`` ie. ``$HOME/.var/app/<app-id>/data/applications``.
```

Reply:
Yea I removed them because Github removes angle brackets when quoting in a comment but I don't see a better solution. So I'll probably add them back.

Reply:
I guess I'll put $FLATPAK_ID there